cfpb/cfgov-refresh
# Feedback form on success behavior

Makes form replacement the default success behavior for feedback forms. Currently some feedback forms are replaced by the success notification (on `owning-a-home/help-us-improve` & Spanish Ask answer pages), but the default behavior (on '/owning-a-home', Ask English answer pages) is to leave the form on the page and add the notification to it. This makes the behavior consistent across the feedback form modules.

## Additions

- Add method to `Notification.js` to get the notification element.

## Changes
- In `feedback.html`
  - Add the block background to the feedback form itself instead of the containing div
  - Move the notification into the feedback form so warning messages appear inside the background
- In `feedback-form.js`
  - Always replace feedback form with success notification
- In `FormSubmit.js`
  - If `replaceForm` option selected, swap form out for its notification module (ensures that notification doesn't appear inside `block__bg`)

## Testing

- Check out branch & run gulp
- Navigate to [Ask answer page](http://localhost:8000/ask-cfpb/how-quickly-can-i-get-money-after-i-deposit-a-check-into-my-checking-account-what-is-a-deposit-hold-en-1023/) and verify that feedback form is replaced on success

## Screenshots
Feedback form error:
<img width="839" alt="screen shot 2017-07-10 at 3 22 29 pm" src="https://user-images.githubusercontent.com/778171/28073115-ae6fffac-6609-11e7-9284-c9ea6edde3f1.png">

Feedback form success:
<img width="792" alt="screen shot 2017-07-10 at 3 23 57 pm" src="https://user-images.githubusercontent.com/778171/28073126-b79baf04-6609-11e7-8769-3a2e607303a3.png">


## Checklist
